### PR TITLE
Add CFI EIPs

### PR DESCRIPTION
**Pull request by timbeiko**
On ACD131, we agreed to move EIPs 3540, 3670 and 3860 to CFI.

### CODE REVIEW

File: network-upgrades/mainnet-upgrades/shanghai.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 ### Included EIPs
 Specifies changes included in the Network Upgrade.
 
+### EIPs Considered for Inclusion
+Specifies changes potentially included in the Network Upgrade, pending successful deployment on Client Integration Testnets.
+
+* [EIP-3540: EVM Object Format (EOF) v1](https://eips.ethereum.org/EIPS/eip-3540)
+* [EIP-3670: EOF - Code Validation](https://eips.ethereum.org/EIPS/eip-3670)
+* [EIP-3860: Limit and meter initcode](https://eips.ethereum.org/EIPS/eip-3860)
```
Comment (axic) [inline network-upgrades/mainnet-upgrades/shanghai.md:12]: I think also 3855 was part of that, at least some client devs referred to it, but would need to listen to the recording for confirmation.

Comment (timbeiko) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: Right, I am actually unclear about this one.

Comment (timbeiko) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: @holiman @MariusVanDerWijden @tkstanczak @yperbasis @shemnon @garyschulte if we could get a 👍 / 👎  from each of your teams on including PUSH0, it'd be helpful so we don't have to wait until the next call.

Comment (shemnon) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: :+1: Implementing PUSH0 is easy, the harder (but still easy) part is ensuring it doesn't leak into pre-Shanghai execution (a well trodden path with ops like chainid). 

The real work is in reference tests and upstream use by solidity and friends.

That extends to all EVM work, good reference tests are more work than client implementations typically.

Comment (axic) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: For the record, Ipsilon has tests for all these EIPs: https://github.com/ethereum/tests/pulls/gumb0

Comment (yperbasis) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: 👍 for [EIP-3855](https://eips.ethereum.org/EIPS/eip-3855) from Erigon.

Comment (MariusVanDerWijden) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: :+1: from geth on PUSH0

Comment (timbeiko) [reply network-upgrades/mainnet-upgrades/shanghai.md:12]: Thanks all for the rapid responses! Will add and merge.

**Pull request merged**